custom_components/tuya_local/devices/klarstein_wonderwall_smart.yaml

```diff
@@ -0,0 +1,54 @@
+name: Klarstein Wonderwall Smart
+products:
+  - id: iuahvrtxktwuuyaf
+    name: Wonderwall Smart
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      name: hvac_mode
+      type: boolean
+      mapping:
+        - dps_val: true
+          value: "heat"
+          icon: "mdi:radiator"
+        - dps_val: false
+          value: "off"
+          icon: "mdi:radiator-disabled"
+    - id: 2
+      name: temperature
+      type: integer
+      mapping:
+        - scale: 1
+          step: 1
+      range:
+        min: 0
+        max: 37
+    - id: 3
+      name: current_temperature
+      type: integer
+      mapping:
+        - scale: 1
+secondary_entities:
+  - entity: number
+    name: Timer
+    icon: "mdi:timer-sand"
+    category: config
+    dps:
+      - id: 10
+        type: integer
+        name: value
+        unit: hour
+        range:
+          min: 0
+          max: 1440
```

Comment:
Is this really the range for dp 10?

This is the only difference between this config and the config I wrote for issue #1272, where dp 10 is a string ranging from "0" to "24" (signifying the number of hours the timer is set for).

Reply:
File removed in deference to klarstein_wonderwall_irheater.yaml added under issue #1272

(it should have been 0-24)